data/yam/agama/auto/sle_nvme.jsonnet

```diff
@@ -0,0 +1,71 @@
+{
+  product: {
+    id: '{{AGAMA_PRODUCT_ID}}',
+  },
+  user: {
+    fullName: 'Bernhard M. Wiedemann',
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true,
+    userName: 'bernhard',
+  },
+  root: {
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true,
+  },
+  storage: {
+    "guided": {
```

Comment:
you cannot clone this, `guided` is deprecated, should not be used, what about if you don't add any storage section, is it done out of the box like in the interactive one? perhaps can be crafted at the moment, not sure, in theory is just another disk.

Reply:
Ok, I removed the storage section for nvme, now just set HDDMODEL=nvme for the test. Updated code and VR.